src/plugin/ipc/socket/socketconnection.cpp

```diff
@@ -221,6 +235,15 @@ bool TcpConnection::isBlacklistedTcp(const sockaddr* saddr, socklen_t len)
     int blacklistedRemotePorts[] = {53,                 // DNS Server
                                     389, 636,           // LDAP
                                     -1};
+#ifdef MPISPAWN_WORKAROUND
```

Comment:
Thinking again, it'll be a performance overhead unless we can move it to a global constructor. The cost is minor but can quickly add up for an application that uses many sockets.

I'll keep it as is for now.